harley/schema_helpers.py

```diff
@@ -0,0 +1,20 @@
+from typing import OrderedDict
+from polars import DataType
+
+def print_schema_as_code(schema: OrderedDict[str, DataType], prepend_pl: bool = False)-> str:
```

Comment:
Might want to be `-> None` as it's printing the result

Reply:
Good shout, implemented in latest commits.